### PR TITLE
Improved User Guide

### PR DESCRIPTION
**Pull request by martinedoesgis**
Hello Qfield team,

I'm following your project for some time and the release of the v1 convinced me to try to contribute to it. My first attempt is by improving a bit the documentation that is sometimes not up-to-date with the last version and that I find a bit sparse and unorganized for now (sorry ... but I know that it's not always the funniest part of a project ;) )

I cleaned and improved with screenshots from the last version the **user guide** and planned to continue working on it later (especially adding a part on editing existing geometries because I really like how you made it user-friendly and the Portable Project and QfieldSync Plugin part with screenshots and concrete example of different possible options). 

Thanks again for this really exciting app ;)

### CODE REVIEW

**Comment by m-kuhn** [comment]
Thanks a lot for this, Martine, it is much appreciated!

I'll have a look and provide feedback as soon as possible. Indeed the documentation is a bit of a shortcoming, so this kind of contribution is very valuable. :cake:

**Comment by martinedoesgis** [comment]
Hello Matthias,

Thanks for your quick answer, it's my pleasure to be able to contribute ;)  

I have two quick questions for you:

- I'm not familiar with Sphinx, so I just wanted to know if the links menu on the left of the documentation page (https://www.qfield.org/docs/en/) is automatically created from the files in GitHub or you do the ordering manually ? If it's automatic, is there a specific syntax to order it ?

- Do you prefer to have the screenshot in the documentation done with a tablet or a smartphone ?

File: en/user-guide/index.rst [hunk 1]
```diff
@@ -50,40 +91,57 @@ To use this you have to `enable mock locations on your Android device
 <https://www.youtube.com/watch?v=v1eRHmMiRJQ>`_.
 
 
-Digitize
+Digitalizing
```
Comment (m-kuhn) [inline en/user-guide/index.rst]:
```suggestion
Digitzing
```

The QGIS documentation uses digitize consistently, I think we should stick to that.

File: en/user-guide/index.rst [hunk 2]
```diff
@@ -50,40 +91,57 @@ To use this you have to `enable mock locations on your Android device
...
+.. image:: ../images/user-guide_digit3.png
 
 Attribute form
 --------------
 
-After digitizing a geometry, the attribute form will be displaied if it is not suppressed
-and the user will be asked to enter the attributes for the new feature.
-The form which appears allows entering attribute values for the new feature. The checkboxes
-at the right of every attribute allow for remembering each attribute individually.
+After digitalizing a geometry, the attribute form will be displayed and the user will be asked to enter the attributes for the new feature.
```
Comment (m-kuhn) [inline en/user-guide/index.rst]:
```suggestion
After digitizing a geometry, the attribute form will be displayed and the user will be asked to enter the attributes for the new feature.
```

**Comment by m-kuhn** [comment]
Hi @martinedoesgis 
I hope you are doing well.
I have left two small suggestions to improve this pull request. If you are ok with them, can you accept them and we can merge this pull request?
It's a really nice enhancement!

**Comment by m-kuhn** [comment]
Hello @martinedoesgis 

The update for the qfield documentation was just integrated. It's an amazing piece of work.

Thanks again and will be nice to hear from you again!

**Pull request merged**